lisa/src/main/java/it/unive/lisa/program/cfg/CFGDescriptor.java

```diff
@@ -414,4 +425,22 @@ public boolean matchesSignature(CFGDescriptor reference) {
 	public CodeLocation getLocation() {
 		return location;
 	}
+
+	/**
+	 * Yields the annotations of this descriptor.
+	 * 
+	 * @return the annotations of this descriptor
+	 */
+	public Annotations getAnnotations() {
+		return annotations;
+	}
+
+	/**
+	 * Adds an annotations to this descriptor.
+	 * 
+	 * @param ann the annotation to be added
+	 */
+	public void addAnnotation(Annotation ann) {
```

Comment:
In some classes we have `addAnnotation`, in some other we have `setAnnotations`. We should unify them